### PR TITLE
feature(ft-sdk): document change for single tester

### PR DESCRIPTION
**Pull request by fivetran-abdulsalam**
Links: T-749376

### Description
Updated tester document with single tester details.
Below CLI args are necessary to run the tester
* --port 
* --tester-type

### CODE REVIEW

File: tools/destination-tester/README.md [hunk 1]
```diff
@@ -145,9 +145,6 @@ Here is an example input file named `input_1.json`:
 
 The tester supports the following optional CLI arguments to alter its default behavior. You can append these options to the end of the `docker run` command provided in step 2 of [How To Run](https://github.com/fivetran/fivetran_sdk/tree/main/tools/destination-tester#how-to-run) section above.
 
-#### --port
```
Comment (varundhall) [inline tools/destination-tester/README.md:148]: why are we removing this? Maybe change the description as --port is still a valid argument

File: tools/connector-tester/README.md [hunk 2]
```diff
@@ -26,8 +26,5 @@ docker start -i <container-id>
 
 The tester supports the following optional CLI arguments to alter its default behavior. You can append these options to the end of the docker run command provided in step 2 of [How To Run](https://github.com/fivetran/fivetran_sdk/blob/main/tools/connector-tester/README.md#how-to-run) section above.
 
-#### --port
```
Comment (varundhall) [inline tools/connector-tester/README.md:29]: same

**Review by varundhall — CHANGES_REQUESTED** [review]

**Comment by varundhall** [comment]
@fivetran-abdulsalam please add a description in the PR.

File: tools/connector-tester/README.md (hunk 2 above)
Comment (fivetran-abdulsalam) [reply tools/connector-tester/README.md:29]: added back port cli with modified description

File: tools/destination-tester/README.md (hunk 1 above)
Comment (fivetran-abdulsalam) [reply tools/destination-tester/README.md:148]: We have added --port in docker command, so removed it. Added back port cli with modified description

**Review by varundhall — APPROVED** [review]

File: tools/connector-tester/README.md [hunk 3]
```diff
@@ -27,7 +27,7 @@ docker start -i <container-id>
 The tester supports the following optional CLI arguments to alter its default behavior. You can append these options to the end of the docker run command provided in step 2 of [How To Run](https://github.com/fivetran/fivetran_sdk/blob/main/tools/connector-tester/README.md#how-to-run) section above.
 
 #### --port
-This option tells the tester to use a different port than the default 50051.
+This option tells the tester on which port it should run.
```
Comment (5tran-alexil) [inline tools/connector-tester/README.md:30]:
```suggestion
This option defines the port the tester should run on.
```

File: tools/destination-tester/README.md [hunk 4]
```diff
@@ -146,7 +146,7 @@ Here is an example input file named `input_1.json`:
 The tester supports the following optional CLI arguments to alter its default behavior. You can append these options to the end of the `docker run` command provided in step 2 of [How To Run](https://github.com/fivetran/fivetran_sdk/tree/main/tools/destination-tester#how-to-run) section above.
 
 #### --port
-This option tells the tester to use a different port than the default 50052.
+This option tells the tester on which port it should run.
```
Comment (5tran-alexil) [inline tools/destination-tester/README.md:149]:
```suggestion
This option defines the port the tester should run on.
```

**Review by 5tran-alexil — APPROVED** [review]
LGTM

**Pull request merged**